Image analysis CoP (#2802)
galaxyproject/galaxy-hub: content/community/sig/imaging/index.md

---
description: "If you need to extract quantitative information from images and want to make your workflows reusable, this community is for you." 
autotoc: true

> title: "iMAGE-NET – Image Analysis Network in Galaxy"

Reviewer: let's discuss this in the broader group

Reviewer 2: Are we changing it to: Galaxy Image Network (GIN)🍸?

Reviewer: I'm collecting here some arguments I've heard against it over lunch:
- Network can be confused with neural network for image analysis
- Image analysis doesn't include e.g. image processing
- To be consistent with the other communities, it would be good to include the word "community" there

> title: "iMAGE-NET – Image Analysis Network in Galaxy"

Reviewer 2: Thanks lots @kostrykin, I'm glad you like the title! I didn’t realize that 'Image-Net' already exists. I prefer names that are easy for people to remember or that create a catchy acronym—like the single-cell community, which is known as SPOC. The minuscule "i" I feel would look good on a logo but we can change that. 

Suggestion for a change could be: GIN – Galaxy Image Network

cc: @beatrizserrano

---

<slot name="/community/sig/common_linkbox" />

# Welcome to iMAGE-NET – Image Analysis Network in Galaxy!

Image analysis bridges multiple disciplines, and this community gathers researchers and developers from diverse scientific fields to share knowledge, tools, and workflows.

## Community goals

Our primary goals are to:
* **Coordinate efforts** across the Galaxy ecosystem to reduce redundancy in tools, workflows, and training

> * **Share our work**—from within and outside Galaxy—to showcase our advancements and inspire new ideas

Reviewer 3: ```suggestion
* **Share our work** — from within and outside Galaxy — to showcase our advancements and inspire new ideas
```
or
```suggestion
* **Share our work** from within and outside Galaxy to showcase our advancements and inspire new ideas
```

* **Support each other** through knowledge-sharing, troubleshooting, and collaboration

Additionally, we strive to:
* **Develop and maintain** image analysis tools within Galaxy
* **Implement FAIR principles** to make workflows reproducible and reusable
* **Expand documentation** and **training resources** for accessible learning and skill-building
  
> *Include I2K Poster https://zenodo.org/records/13847465 *

## Who We Are

We are a diverse group of users, developers, trainers, and imaging data enthusiasts, collaborating to expand Galaxy’s image analysis capabilities, bringing the latest tools, feedback, and beyond.

> - **Developers**: Are you building image analysis tools in Galaxy, or thinking of starting a project here? Do you need users, testers, or just want to make sure you’re not duplicating tools that already exist? Join us to connect, share ideas, and get support for your development journey!

Reviewer 3: ```suggestion
- **Developers**: Are you building image analysis tools in Galaxy, or thinking of starting a project here? Do you need users, testers, or just want to make sure you're not duplicating tools that already exist? Join us to connect, share ideas, and get support for your development journey!
```

> - **Users**: Do you use Galaxy’s image analysis tools for bio-imaging, Earth observation, or other fields? Have feedback for us, or interested in testing new resources? Perhaps you have ideas or requests for new tools—your input can help shape future developments!

Reviewer 3: ```suggestion
- **Users**: Do you use Galaxy's image analysis tools for bio-imaging, Earth observation, or other fields? Have feedback for us, or interested in testing new resources? Perhaps you have ideas or requests for new tools — your input can help shape future developments!
```

> - **Trainers**: Are you using Galaxy to train others in image analysis? If you have suggestions or specific needs to better support your trainees, we’d love to hear from you. Your insights are invaluable in helping us create better learning experiences and share data image analysis knowledge. 

Reviewer 3: ```suggestion
- **Trainers**: Are you using Galaxy to train others in image analysis? If you have suggestions or specific needs to better support your trainees, we'd love to hear from you. Your insights are invaluable in helping us create better learning experiences and share data image analysis knowledge. 
```

# Getting Started: GTN Tutorials

To dive into Galaxy, the [Galaxy Training Network (GTN)](https://training.galaxyproject.org) offers a wealth of tutorials for beginners and experts alike. These tutorials cover basic and advanced workflows, guiding you through analyses step-by-step. They also include a curated selection of workflows to help you gain hands-on experience in Galaxy.

> *Embed the GTN tutorial table*

# What is available?

## Image Analysis Tools

The Galaxy ToolShed offers a variety of tools for image data analysis, freely available for installation on any Galaxy server. Our community maintains a curated list of available tools to keep users informed and up-to-date.

<iframe
  id="inlineFrameExample"
  title="Image analysis related tools"
  width="100%"
  height="600"
  frameBorder="0"
  src="https://galaxyproject.github.io/galaxy_codex/imaging/">
</iframe>

> *Is something missing? Reach out to us to keep the list comprehensive.*

## Interactive Environments

> Image analysis often involves hands-on interaction, and Galaxy offers resources that make this possible. Check out our interactive tools to experience Galaxy’s potential beyond traditional workflows.

Reviewer 3: ```suggestion
Image analysis often involves hands-on interaction, and Galaxy offers resources that make this possible. Check out our interactive tools to experience Galaxy's potential beyond traditional workflows.
```

## Visualizations

The **zarr** format is becoming a standard for storing large image data. With various adaptations like OME-Zarr and GEO-Zarr, this interoperable format enhances data reusability across disciplines. Our community supports zarr-based visualization through tools like vizarr, helping bring this dream of cross-discipline interoperability to life.

> *Include vizarr visualization here.*

## A Dedicated Galaxy Interface

> Our dedicated interface for image data analysis—[imaging.usegalaxy.eu](https://imaging.usegalaxy.eu)—provides a comprehensive suite of tools and workflows tailored specifically for imaging scientists.

Reviewer 3: Maybe I'm missing something here, but how is imaging.usegalaxy.eu a user interface? It's more like an entry page, isn't it?

> *Include a screenshot of the dedicated interface for better visual context.*

# How is Galaxy used for image data analysis?

## Projects / Showcases

## Partner Projects

Community members actively pursue funding opportunities and contribute to various projects, such as:

| Project Name      | Description                                                                                           | Community Members Involved                                | Funding Period |
|-------------------|-------------------------------------------------------------------------------------------------------|-----------------------------------------------------------|----------------|
| **NFDI4BIOIMAGE** | Developing open and FAIR workflows for biological image data                                          | Riccardo ...                                              | 2023 - XXX     |
| [**OSCARS**](https://oscars-project.eu/) | Creating reusable workflows for diverse disciplines like microscopy, astrophysics, and environmental sciences | Beatriz Serrano-Solano, Anne Fouilloux, Matus Kalas, ...  | 01.2025 - 01.2027    |

# How to get involved?

> If you’re interested in image analysis (within or outside Galaxy) and want to help ensure our tools represent the best, up-to-date practices, your feedback is needed! We’re constantly looking for ways to improve, whether in scientific fields already supported by Galaxy image analysis or in exploring new applications.

Reviewer 3: ```suggestion
If you're interested in image analysis (within or outside Galaxy) and want to help ensure our tools represent the best, up-to-date practices, your feedback is needed! We're constantly looking for ways to improve, whether in scientific fields already supported by Galaxy or in exploring new applications.
```

> If you’re passionate about shaping the future of image analysis in Galaxy and want to help build a strong, collaborative, and supportive community, **you’re in the right place!**

Reviewer 3: ```suggestion
If you're passionate about shaping the future of image analysis in Galaxy and want to help build a strong, collaborative, and supportive community, **you're in the right place!**
```

<img class="img-fluid float-left" src="/community/sig/image-analysis/FAIR-wf-postcard.png" style="width:450px;" alt="FAIR Workflows Expert Group postcard"/>

- **Frequency**: Monthly
- **Duration**: 1 hour
- **Time**: 3rd Wednesday of the month at 4 pm CET
- **Meeting Agenda**: Updates from the community (e.g., upcoming events), relevant talks or case studies, and coordination for activities such as grant writing, paper development, and event planning. View our open agenda and notes here.

> *Consider adding calendar invite like this for easy scheduling:*

<iframe src="https://calendar.google.com/calendar/embed?height=300&wkst=1&bgcolor=%23ffffff&ctz=UTC&showTitle=1&showNav=1&mode=AGENDA&showDate=1&src=bnB1N3B1azc1bjZwMHU0YWFwdjV0MjF1ajhAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&color=%23AD1457" style="border:solid 1px #777" width="800" height="300" frameborder="0" scrolling="no"></iframe>

In-person meetups are also part of our community engagement. We recently participated in BioHackathon 2023 and 2024, where we collaborated on new projects and shared preprints.

# Acknowledgments

> Our community’s success is made possible through partnerships and collaborations. We thank our partners for their support.

Reviewer 3: ```suggestion
Our community's success is made possible through partnerships and collaborations. We thank our partners for their support.
```
